EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFSetup.cc

```diff
@@ -50,9 +50,6 @@ namespace l1t {
       UnpackerMap
       BMTFSetup::getUnpackers(int fed, int board, int amc, unsigned int fw)
       {
-         auto outputMuon = UnpackerFactory::get()->make("stage2::BMTFUnpackerOutput");
-         auto inputMuons = UnpackerFactory::get()->make("stage2::BMTFUnpackerInputs");
-
          UnpackerMap res;
```

Comment:
@matz-e there is still a duplication here. The UnpackerMap variable declaration and the for-loop appear again in line 63.